contrib/drivers/lvm/metricsCLI.go

```diff
@@ -0,0 +1,99 @@
+package lvm
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+	myExec "os/exec"
+)
```

Comment:
Same as above

Reply:
Please change the file name to `metrics_cli.go`.

Reply:
renamed